src/mozillavpn.cpp

```diff
@@ -449,6 +449,12 @@ void MozillaVPN::openLink(LinkType linkType) {
       url = NetworkRequest::apiBaseUrl();
       url.append("/r/vpn/subscriptionBlocked");
       break;
+    case LinkSplitTunnelHelp:
+      // TODO: This should link to a more helpful article
```

Comment:
@lesleyjanenorton can you help finding the final url?

Reply:
Sure!

Reply:
Seems like `https://support.mozilla.org/en-US/kb/split-tunneling-use-mozilla-vpn-specific-apps-wind` is final but needs to be updated.

Reply:
Running this by Brian to see how we can get something better and will follow back up.